wear/src/main/java/io/homeassistant/companion/android/complications/views/ComplicationConfigMainView.kt

```diff
@@ -133,54 +131,44 @@ fun MainConfigView(
             }
             item {
                 val isChecked = !loaded || showTitle
-                ToggleChip(
+                val description = getDescription(isChecked)
+                ToggleButton(
                     checked = isChecked,
                     onCheckedChange = onShowTitleClicked,
-                    label = {
-                        Text(
-                            stringResource(R.string.show_entity_title),
-                            fontWeight = FontWeight.Bold
-                        )
-                    },
-                    toggleControl = {
-                        Icon(
-                            imageVector = ToggleChipDefaults.switchIcon(isChecked),
-                            contentDescription = if (isChecked) {
-                                stringResource(R.string.enabled)
-                            } else {
-                                stringResource(R.string.disabled)
+                    label = { Text(stringResource(R.string.show_entity_title)) },
+                    selectionControl = {
+                        Switch(
+                            checked = isChecked,
+                            modifier = Modifier.semantics {
+                                this.contentDescription = description
                             },
-                            tint = if (isChecked) wearColorScheme.tertiary else wearColorScheme.onSurface
+                            colors = switchColors()
                         )
                     },
                     modifier = Modifier.fillMaxWidth(),
-                    enabled = loaded && entity != null
+                    enabled = loaded && entity != null,
+                    colors = ToggleButtonDefaults.toggleButtonColors(checkedContainerColor = wearColorScheme.outlineVariant)
                 )
             }
             item {
                 val isChecked = !loaded || showUnit
-                ToggleChip(
+                val description = getDescription(isChecked)
+                ToggleButton(
                     checked = isChecked,
                     onCheckedChange = onShowUnitClicked,
-                    label = {
-                        Text(
-                            stringResource(R.string.show_unit_title),
-                            fontWeight = FontWeight.Bold
-                        )
-                    },
-                    toggleControl = {
-                        Icon(
-                            imageVector = ToggleChipDefaults.switchIcon(isChecked),
-                            contentDescription = if (isChecked) {
-                                stringResource(R.string.enabled)
-                            } else {
-                                stringResource(R.string.disabled)
+                    label = { Text(stringResource(R.string.show_unit_title)) },
+                    selectionControl = {
+                        Switch(
```

Comment:
This seems quite standard, so maybe it's worth putting the entire `Switch` with checked, modifier and colors in a Composable?

Reply:
ah good point :)